tests/components/websocket_api/test_commands.py

```diff
@@ -701,7 +702,9 @@ async def test_get_services(
         assert msg["id"] == id_
         assert msg["type"] == const.TYPE_RESULT
         assert msg["success"]
-        assert msg["result"] == hass.services.async_services()
+
+        descriptions = await async_get_all_descriptions(hass)
```

Comment:
These changes don't seem related to the newly added service.
Should probably best be split off into a separate PR.

Reply:
Hmm yes, would indeed be better, even if this was noticed and needed by this change.

Before the PR, `hass.services.async_services()` returned an empty dict as the components being loaded in those tests (e.g. http) had no services. Since we now have a service in HTTP, the comparison fails because `msg["result"]` is a JSON representation and `hass.services.async_services()` is a Object representation